File: mesh-delivery/android/ExoPlayerMeshDeliveryJava/app/src/main/java/io/streamroot/lumen/delivery/client/samples/mesh_delivery/exoplayer/Application.java
package io.streamroot.lumen.delivery.client.samples.mesh_delivery.exoplayer;

import androidx.multidex.MultiDexApplication;

import io.streamroot.lumen.delivery.client.core.LumenDeliveryClient;

public class Application extends MultiDexApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        LumenDeliveryClient.initializeApp(this);
    }
}

File: mesh-delivery/android/ExoPlayerMeshDeliveryPlugin/app/src/main/java/io/streamroot/lumen/delivery/client/samples/mesh_delivery/exoplayer/Application.kt
package io.streamroot.lumen.delivery.client.samples.mesh_delivery.exoplayer

import androidx.multidex.MultiDexApplication
import io.streamroot.lumen.delivery.client.core.LumenDeliveryClient

class Application : MultiDexApplication() {
    override fun onCreate() {
        super.onCreate()
        LumenDeliveryClient.initializeApp(this)
    }
}